File: Java/UberVan.java
package Java;

import java.util.ArrayList;
import java.util.Map;

public class UberVan extends Car {
    Map<String, Map<String,Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;

    public UberVan(String l, Account d){
        super(l, d);
    }

    @Override
    public void setPassenger(Integer p) {
        if(p == 6){
            this.passenger = p;
        } else {
            System.out.println("Necesitas asignar 6 pasajeros");
        }
    }
}

File: Python/uberVan.py
from car import Car

class UberVan(Car):
    typeCarAccepted = []
    seatsMaterial = []
    
    def __init__(self, l, d, tCA, sM):
        super().__init__(l, d)
        self.typeCarAccepted = tCA
        self.seatsMaterial = sM